cookbooks/hangops-jobbot/recipes/default.rb

```diff
@@ -14,6 +14,12 @@
 # limitations under the License.
 #
 
+# Create a Hubot user
+user 'hubot' do
+   action :create #default action
+   group "groupid"
```

Comment:
yeah, that's fine if the group "groupid" exists. If not, @rrxtns should update the group "groupid" to group "robots" or something.